src/map/packets/quest_mission_log.h

```diff
@@ -25,66 +25,73 @@
 #define _CQUESTMISSIONLOGPACKET_H
 
 #include "../../common/cbasetypes.h"
+#include <unordered_map>
 
 #include "basic.h"
 
-//Quest Log Types
-#define SAN_CURRENT			0x50
-#define BAS_CURRENT			0x58
-#define WIN_CURRENT			0x60
-#define JEU_CURRENT			0x68
-#define OTH_CURRENT			0x70
-#define OUT_CURRENT			0x78
-#define EXP_CURRENT			0x80
-#define WAR_CURRENT			0x88
-#define ABY_CURRENT			0xE0
-#define ADO_CURRENT			0xF0
-#define COA_CURRENT			0x0100
-
-#define SAN_COMPLETE		0x90
-#define BAS_COMPLETE		0x98
-#define WIN_COMPLETE		0xA0
-#define JEU_COMPLETE		0xA8
-#define OTH_COMPLETE		0xB0
-#define OUT_COMPLETE		0xB8
-#define EXP_COMPLETE		0xC0
-#define WAR_COMPLETE		0xC8
-#define ABY_COMPLETE		0xE8
-#define ADO_COMPLETE		0xF8
-#define COA_COMPLETE		0x0108
-
-//Mission Log Types
-#define MISS_COMPLETE		0xD0
-#define MISS_CURRENT		0xFFFF
-
-// MISS_CURRENT	0xFF исключая Add-on Scenarios
-
-#define EXP_MISS_COMPLETE               0xD8
-#define EXP_MISS_CURRENT                0x80
-
-#define CAMPAIGN_MISSION_UN             0x30
-#define CAMPAIGN_MISSION_DEUX           0x38
-
-// Mission Logs
-#define MISSION_SANDORIA                0x0B
-#define MISSION_BASTOK                  0x0C
-#define MISSION_WINDURST                0x0D
-#define MISSION_ZILART                  0x0E
-#define MISSION_TOAU                    0x0F
-#define MISSION_WOTG                    0x10
-#define MISSION_COP                     0x11
-
-#define MISSION_ASSAULT                 0x12
-
-#define MISSION_CAMPAIGN                0x13
-#define MISSION_CAMPAIGN2               0x14
-
-#define MISSION_ACP                     0x15
-#define MISSION_AMK		                0x16
-#define MISSION_ASA		                0x17
-
-#define MISSION_SOA					    0x18
-#define MISSION_ROV                     0x19
+// Quest/Mission Area Identifiers
+#define QUESTS_SANDORIA      0
+#define QUESTS_BASTOK        1
+#define QUESTS_WINDURST      2
+#define QUESTS_JEUNO         3
+#define QUESTS_OTHER         4
+#define QUESTS_OUTLANDS      5
+#define QUESTS_TOAU          6
+#define QUESTS_WOTG          7
+#define QUESTS_ABYSSEA       8
+#define QUESTS_ADOULIN       9
+#define QUESTS_COALITION    10
+
+#define MISSION_SANDORIA     0
+#define MISSION_BASTOK       1
+#define MISSION_WINDURST     2
+#define MISSION_ZILART       3
+#define MISSION_TOAU         4
+#define MISSION_WOTG         5
+#define MISSION_COP          6
+#define MISSION_ASSAULT      7
+#define MISSION_CAMPAIGN     8
+#define MISSION_ACP          9
+#define MISSION_AMK         10
+#define MISSION_ASA         11
+#define MISSION_SOA         12
+#define MISSION_ROV         13
+
+// Log Types
+enum LOG_TYPE
+{
+    LOG_QUEST_CURR = 1,
+    LOG_QUEST_COMP = 2,
+    LOG_MISS_CURR = 3,
+    LOG_MISS_COMP = 4,
```

Comment:
LOG_MISSION_COMPLETE etc
need to get rid of the shorthand enums at some point (like for skills)